Sources/CoreCommands/SwiftCommandState.swift

```diff
@@ -872,11 +876,14 @@ public final class SwiftCommandState {
     /// Lazily compute the host toolchain used to compile the package description.
     private lazy var _hostToolchain: Result<UserToolchain, Swift.Error> = {
         return Result(catching: {
-            try UserToolchain(
-                swiftSDK: SwiftSDK.hostSwiftSDK(
-                    environment: self.environment,
-                    observabilityScope: self.observabilityScope
-                ),
+            var hostSwiftSDK = try SwiftSDK.hostSwiftSDK(
+                environment: self.environment,
+                observabilityScope: self.observabilityScope
+            )
+            hostSwiftSDK.targetTriple = self.hostTriple
```

Comment:
Not for this PR - maybe we should rename this to `triple`?

Reply:
On `SwiftSDK` or on `SwiftCommandState`?

Reply:
SwiftSDK

Reply:
SwiftSDKs have both triples specified on purpose, as one can have a Swift SDK installed supporting certain host triples that don't include the triple of a machine it's installed on.